runtime/oti/j9nonbuilder.h

```diff
@@ -3218,9 +3218,12 @@ typedef struct J9ClassLoader {
 #define J9CLASSLOADER_SET_CLASSLOADEROBJECT(vmThread, object, value) J9VMTHREAD_JAVAVM(vmThread)->memoryManagerFunctions->j9gc_objaccess_storeObjectToInternalVMSlot((vmThread), (j9object_t*)&((object)->classLoaderObject), (value))
 #define TMP_J9CLASSLOADER_CLASSLOADEROBJECT(object) ((object)->classLoaderObject)
 
+#if defined(_MSC_VER)
+#pragma warning(disable : 4200)
```

Comment:
This should disable the warning only for the following struct declaration:
```
#pragma warning(push)
#pragma warning(disable : 4200)
```
and restore the warning after the declaration:
```
#pragma warning(pop)
```

Reply:
If you would prefer to disable that warning globally (https://github.com/eclipse/openj9/issues/10244#issuecomment-675213567), then this is not the place to do it.

Reply:
I'll make this change for now (note I copied this from OMR, which does not push/pop, so you may want to suggest the same there). We can remove it later if we opt for the global warning disable.

Reply:
OMR isn't consistent: Some places it uses `#pragma warning(default:N)`, some places it says `#pragma warning(pop)` and some places neither appears. I created eclipse/omr#5478 to track improving things.